src/Pages/Home.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Badge from 'react-bootstrap/Badge';
+import ListGroup from 'react-bootstrap/ListGroup';
+import CloseButton from 'react-bootstrap/CloseButton';
+import { fetchStocks } from '../redux/features/stocks/stocksSlice';
+import { fetchDetails } from '../redux/features/details/detailsSlice';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    dispatch(fetchStocks());
+  }, [dispatch]);
+
+  const stocks = useSelector((state) => state.stocks.stocks);
+
+  const [searchQuery, setSearchQuery] = useState('');
+  const filteredStocks = stocks
+    .filter((stock) => stock.symbol.toLowerCase().includes(searchQuery.toLowerCase()));
+
+  const displayedStocks = searchQuery ? filteredStocks : stocks;
+
+  return (
+    <Container>
+      <Row style={{ margin: '1rem 0.1rem', alignItems: 'center' }}>
+        <Col style={{ textAlign: 'center', fontSize: '1rem' }}>
+          <span>
+            <i className="uil uil-angle-left-b" />
+            {' '}
```

Comment:
- [ ] The `<Home />` component is blank and doesn't render anything. This is not the expected behaviour. Please try to fix it and ensure that it adheres to the layout of the design guidelines 🙏 
### Guidelines
![image](https://github.com/Paulinelali/metric-capstone/assets/46839548/6eb256f0-1269-45e4-9715-609510c2576a)
### Your Implementation
![image](https://github.com/Paulinelali/metric-capstone/assets/46839548/381c893d-ebcb-464e-bfec-530ac72d6e89)